tools/serverpod_cli/lib/src/generator/open_api/helpers/utils.dart

```diff
@@ -0,0 +1,148 @@
+/// Converts a list of directory parts to a path string.
+///
+/// Example: ['api', 'v1'] => '/api/v1'
+String getExtraPath(List<String> subDirParts) =>
+    subDirParts.isEmpty ? '' : "/${subDirParts.join('/')}";
+
+/// The location of the parameter. Possible values are "query", "header",
```

Comment:
If these are the viable options, why is there also a `body` in the enum? Incorrect documentation may be worse than no documentation. 😅